main.tf

```diff
@@ -1 +1,11 @@
-#(empty)
+### SCPs module
+resource "aws_organizations_policy" "main" {
+  name    = var.name
```

Comment:
Add to the changelog the hability to add multiple SCPs by either using `count` or `for` loop